src/CSSBattle/10_CloakedSpirits/ZhangZhikai.html

```diff
@@ -0,0 +1,83 @@
+<div class="bgrect">
+  <div class="rectcover"></div>
```

Comment:
Consider using pascalCase for naming

Reply:
Can use VSCode's `Change All Occurrences` to help in this

Reply:
69d8fbf344b344fa08ed6bab1ef8ea25965c5b03